operations/registers.md

```diff
@@ -0,0 +1,97 @@
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
+# Registers
+
+## A register within an outlet.
+
+A register represents a terminal within an outlet, used to issue invoices. Each register is
+uniquely identified within its outlet and can either be physical or virtual.
+Note: This operation needs [Authentication](../essential-guide/authentication.md) and 
+supports the `include` query parameter `outlet` following the [JSON:API specification](../essential-guide/features.md).
+
+### Request
+
+`GET` `[PlatformAddress]/api/v2/registers/{id}`
```

Comment:
Just curious, but are we already on v2?

Reply:
@MikeAdamsMews Yes we are. We already have an internal api v1, so v2 is essentially a redesign and rethink of v1 to support our external partners.